src/components/form/form.scss

```
@@ -0,0 +1,195 @@
+@use 'styles/utils';
+
+.question-form {
+  @include utils.bg-defaults(cover);
+  background-image: url("assets/form-bg.jpg");
+  padding-top: 148px;
+  position: relative;
+
+  &::after {
+    content: '';
+    position: absolute;
+    inset: 0;
+    background-color: hsla(245, 96%, 17%, 0.3);
+  }
+
+  .container {
+    position: relative;
+    z-index: 2;
+  }
+
+  .section-title-wrapper {
+    margin-bottom: 60px;
+  }
+
+  &__wrapper {
+    padding-bottom: 60px;
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-start;
+    gap: 24px;
+  }
+
+  &__content {
+
+    .title {
+      @include utils.text-header-100;
+      margin-bottom: 24px;
+      color: var(--base100);
+    }
+
+    .text {
+      @include utils.text-base-200-lg-100;
+      color: var(--base100);
+      opacity: .5;
+      margin-bottom: 39px;
+    }
+
+    .socials {
+      display: flex;
+      align-items: center;
+      gap: 8px;
+
+      &__img {
+        width: 80px;
+        height: 80px;
+        border-radius: 50%;
+        background-color: hsla(0, 0%, 100%, 0.1);
+        display: flex;
+        align-items: center;
+        justify-content: center;
+
+        img {
+          filter: brightness(10);
+        }
+      }
+    }
+  }
+
+  form {
+    min-width: 753px;
```

Если бы мы не применяли `calc` для выставления значений, то при уменьшении ширины адаптив работал не верно. Для верной адаптивности нужно использовать `max-width` с `width`

```suggestion
    max-width: 753px;
    width: 100%;
```

---

исправила